.CI/common.groovy

```diff
@@ -1,13 +1,29 @@
+
+def isWindows
+{
+  return !isUnix()
+}
+
 void standardSetup() {
   echo "${env.NODE_NAME}"
   // Jenkins cleans with -fdx; --ffdx is needed to remove git repositories
-  sh "git clean -ffdx && git submodule foreach --recursive git clean -ffdx"
+  sh "git clean -ffdx -e OMSetup && git submodule foreach --recursive git clean -ffdx"
 }
 
 def numPhysicalCPU() {
   if (env.JENKINS_NUM_PHYSICAL_CPU) {
     return env.JENKINS_NUM_PHYSICAL_CPU
   }
+  
+  if isWindows()
+  {
+    return sh (
+      script: 'echo %NUMBER_OF_PROCESSORS%',
+      returnStdout: true
+    ).trim().toInteger() / 2 ?: 1
```

Comment:
Couldn't you just do `env.NUMBER_OF_PROCESSORS.toInteger() / 2 ?: 1`

Reply:
most likely, i didn't know that worked :)

Reply:
i'll fix it :)